Over a thousand warnings when building coreclr

Building coreclr now produces over a thousand warnings.  Most if not all of them appear to be non-CLS-compliance warnings stemming from the recently merged intrinsics work, e.g.
```
...
src\System\Runtime\Intrinsics\X86\Avx2.cs(1326,90): warning CS3001: Argument type 'Vector256<ulong>' is not CLS-complia
nt [c:\Users\stoub\Source\Repos\coreclr\src\mscorlib\System.Private.CoreLib.csproj]
src\System\Runtime\Intrinsics\X86\Avx2.cs(1359,40): warning CS3002: Return type of 'Avx2.Xor(Vector256<ulong>, Vector25
6<ulong>)' is not CLS-compliant [c:\Users\stoub\Source\Repos\coreclr\src\mscorlib\System.Private.CoreLib.csproj]
src\System\Runtime\Intrinsics\X86\Avx2.cs(1331,40): warning CS3002: Return type of 'Avx2.Xor(Vector256<sbyte>, Vector25
6<sbyte>)' is not CLS-compliant [c:\Users\stoub\Source\Repos\coreclr\src\mscorlib\System.Private.CoreLib.csproj]
src\System\Runtime\Intrinsics\X86\Avx2.cs(1359,61): warning CS3001: Argument type 'Vector256<ulong>' is not CLS-complia
nt [c:\Users\stoub\Source\Repos\coreclr\src\mscorlib\System.Private.CoreLib.csproj]
src\System\Runtime\Intrinsics\X86\Avx2.cs(1331,61): warning CS3001: Argument type 'Vector256<sbyte>' is not CLS-complia
nt [c:\Users\stoub\Source\Repos\coreclr\src\mscorlib\System.Private.CoreLib.csproj]
src\System\Runtime\Intrinsics\X86\Avx2.cs(1359,84): warning CS3001: Argument type 'Vector256<ulong>' is not CLS-complia
nt [c:\Users\stoub\Source\Repos\coreclr\src\mscorlib\System.Private.CoreLib.csproj]
src\System\Runtime\Intrinsics\X86\Avx2.cs(1331,84): warning CS3001: Argument type 'Vector256<sbyte>' is not CLS-complia
nt [c:\Users\stoub\Source\Repos\coreclr\src\mscorlib\System.Private.CoreLib.csproj]
    1192 Warning(s)
```

--- a/src/mscorlib/src/System/Runtime/Intrinsics/X86/Aes.cs
+++ b/src/mscorlib/src/System/Runtime/Intrinsics/X86/Aes.cs
@@ -10,6 +10,7 @@ namespace System.Runtime.Intrinsics.X86
     /// <summary>
     /// This class provides access to Intel AES hardware instructions via intrinsics
     /// </summary>
+    [CLSCompliant(false)]
     public static class Aes 
     {
         public static bool IsSupported { get { return false; } }

--- a/src/mscorlib/src/System/Runtime/Intrinsics/X86/Avx.cs
+++ b/src/mscorlib/src/System/Runtime/Intrinsics/X86/Avx.cs
@@ -10,6 +10,7 @@ namespace System.Runtime.Intrinsics.X86
     /// <summary>
     /// This class provides access to Intel AVX hardware instructions via intrinsics
     /// </summary>
+    [CLSCompliant(false)]
     public static class Avx
     {
         public static bool IsSupported { get { return false; } }

--- a/src/mscorlib/src/System/Runtime/Intrinsics/X86/Avx2.cs
+++ b/src/mscorlib/src/System/Runtime/Intrinsics/X86/Avx2.cs
@@ -10,6 +10,7 @@ namespace System.Runtime.Intrinsics.X86
     /// <summary>
     /// This class provides access to Intel AVX2 hardware instructions via intrinsics
     /// </summary>
+    [CLSCompliant(false)]
     public static class Avx2
     {
         public static bool IsSupported { get { return false; } }

--- a/src/mscorlib/src/System/Runtime/Intrinsics/X86/Bmi1.cs
+++ b/src/mscorlib/src/System/Runtime/Intrinsics/X86/Bmi1.cs
@@ -10,6 +10,7 @@ namespace System.Runtime.Intrinsics.X86
     /// <summary>
     /// This class provides access to Intel BMI1 hardware instructions via intrinsics
     /// </summary>
+    [CLSCompliant(false)]
     public static class Bmi1
     {
         public static bool IsSupported { get { return false; } }

--- a/src/mscorlib/src/System/Runtime/Intrinsics/X86/Bmi2.cs
+++ b/src/mscorlib/src/System/Runtime/Intrinsics/X86/Bmi2.cs
@@ -10,6 +10,7 @@ namespace System.Runtime.Intrinsics.X86
     /// <summary>
     /// This class provides access to Intel BMI2 hardware instructions via intrinsics
     /// </summary>
+    [CLSCompliant(false)]
     public static class Bmi2
     {
         public static bool IsSupported { get { return false; } }

--- a/src/mscorlib/src/System/Runtime/Intrinsics/X86/Lzcnt.cs
+++ b/src/mscorlib/src/System/Runtime/Intrinsics/X86/Lzcnt.cs
@@ -10,6 +10,7 @@ namespace System.Runtime.Intrinsics.X86
     /// <summary>
     /// This class provides access to Intel LZCNT hardware instructions via intrinsics
     /// </summary>
+    [CLSCompliant(false)]
     public static class Lzcnt
     {
         public static bool IsSupported { get { return false; } }

--- a/src/mscorlib/src/System/Runtime/Intrinsics/X86/Pclmulqdq.cs
+++ b/src/mscorlib/src/System/Runtime/Intrinsics/X86/Pclmulqdq.cs
@@ -10,6 +10,7 @@ namespace System.Runtime.Intrinsics.X86
     /// <summary>
     /// This class provides access to Intel PCLMULQDQ hardware instructions via intrinsics
     /// </summary>
+    [CLSCompliant(false)]
     public static class Pclmulqdq
     {
         public static bool IsSupported { get { return false; } }

--- a/src/mscorlib/src/System/Runtime/Intrinsics/X86/Popcnt.cs
+++ b/src/mscorlib/src/System/Runtime/Intrinsics/X86/Popcnt.cs
@@ -10,6 +10,7 @@ namespace System.Runtime.Intrinsics.X86
     /// <summary>
     /// This class provides access to Intel POPCNT hardware instructions via intrinsics
     /// </summary>
+    [CLSCompliant(false)]
     public static class Popcnt
     {
         public static bool IsSupported { get { return false; } }

--- a/src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse.cs
+++ b/src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse.cs
@@ -10,6 +10,7 @@ namespace System.Runtime.Intrinsics.X86
     /// <summary>
     /// This class provides access to Intel SSE hardware instructions via intrinsics
     /// </summary>
+    [CLSCompliant(false)]
     public static class Sse
     {
         public static bool IsSupported { get { return false; } }

--- a/src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse2.cs
+++ b/src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse2.cs
@@ -10,6 +10,7 @@ namespace System.Runtime.Intrinsics.X86
     /// <summary>
     /// This class provides access to Intel SSE2 hardware instructions via intrinsics
     /// </summary>
+    [CLSCompliant(false)]
     public static class Sse2
     {
         public static bool IsSupported { get { return false; } }

--- a/src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse3.cs
+++ b/src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse3.cs
@@ -10,6 +10,7 @@ namespace System.Runtime.Intrinsics.X86
     /// <summary>
     /// This class provides access to Intel SSE3 hardware instructions via intrinsics
     /// </summary>
+    [CLSCompliant(false)]
     public static class Sse3
     {
         public static bool IsSupported { get { return false; } }

--- a/src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse41.cs
+++ b/src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse41.cs
@@ -10,6 +10,7 @@ namespace System.Runtime.Intrinsics.X86
     /// <summary>
     /// This class provides access to Intel SSE4.1 hardware instructions via intrinsics
     /// </summary>
+    [CLSCompliant(false)]
     public static class Sse41
     {
         public static bool IsSupported { get { return false; } }

--- a/src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse42.cs
+++ b/src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse42.cs
@@ -10,6 +10,7 @@ namespace System.Runtime.Intrinsics.X86
     /// <summary>
     /// This class provides access to Intel SSE4.2 hardware instructions via intrinsics
     /// </summary>
+    [CLSCompliant(false)]
     public static class Sse42
     {
         public static bool IsSupported { get { return false; } }

--- a/src/mscorlib/src/System/Runtime/Intrinsics/X86/Ssse3.cs
+++ b/src/mscorlib/src/System/Runtime/Intrinsics/X86/Ssse3.cs
@@ -10,6 +10,7 @@ namespace System.Runtime.Intrinsics.X86
     /// <summary>
     /// This class provides access to Intel SSSE3 hardware instructions via intrinsics
     /// </summary>
+    [CLSCompliant(false)]
     public static class Ssse3
     {
         public static bool IsSupported { get { return false; } }

